prow/config.gen.yaml

```diff
@@ -2330,6 +2330,9 @@ presubmits:
         - mountPath: /sys/fs/cgroup
           name: cgroup
           readOnly: true
+        - mountPath: /var/lib/docker/overlay2
```

Comment:
Maybe a comment here as a breadcrumb for those coming after us is helpful as to why these lines were added?

Reply:
I can do that in a follow up (and that applies to all mounts above it)

I'm just trying to see if this works